CHANGELOG.md

```diff
@@ -25,6 +26,115 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v0.7.0
+
+The v0.7.0 release focuses on refining and stabilizing existing APIs. This
+included a focus on both conformance tests and clarifying ambiguous parts of the
+API spec.
+
+## Features Graduating to Standard
+In addition to those broad focuses, 2 features are graduating to the
+standard channel:
+
+* GEP-1323: Response Header Modifiers (#1905, @robscott)
+* GEP-726: Path Redirects and Rewrites (#1905, @robscott)
+
+## GEPs
+There are a lot of interesting GEPs in the pipeline right now, but in v0.7.0
+we're highlighting two experimental GEPs:
+
+### GEP-713: Policy Attachment
+This GEP received a major update, splitting policy attachment into two
+categories "Direct" and "Inherited". The new "Direct" mode enables a simplified
+form of policy attachment for targeting a single resource (#1565, @youngnick).
+
+### GEP-1748: Gateway API Interaction with Multi-Cluster Services
+A new GEP was introduced to define how Gateway API interacts with Multi-Cluster
+Services. At a high level, this states that ServiceImports have "Extended"
+support and can be used anywhere Services can throughout the API. There's a lot
+more nuance here, so for the full details, refer to the GEP. (#1843, @robscott)
+
+## Other Changes by Kind
+
+### Status Changes
+
+- The "Ready" Gateway and Listener condition has been reserved for future use.
+  (#1888, @howardjohn)
+- The UnsupportedAddress Listener condition reason has been moved to a Gateway
+  condition reason.  (#1888, @howardjohn)
+- The AddressNotAssigned Gateway condition reasons has moved from Accepted to
+  Programmed. (#1888, @howardjohn)
+- The NoResources Gateway condition reasons has moved from Ready to Programmed.
+  (#1888, @howardjohn)
+
+### Spec Cleanup
+
+- Clarification that port redirects should not add port number to Location
+  header for HTTP and HTTPS requests on 80 and 443. (#1908, @robscott)
+- Port redirect when empty will depend on the configured Redirect scheme (#1880,
+  @gauravkghildiyal)
+- Updated spec to clarify that Exact matches have precedence over Prefix matches
+  and RegularExpression matches have implementation specific precedence. (#1855,
+  @Xunzhuo)
+- The `gateway-exists-finalizer.gateway.networking.k8s.io` finalizer is no
+  longer required and is now just recommended. (#1917, @howardjohn)
+
+### Validation Fixes
+
+- Removes GRPCRoute method match defaulting to allow for matching all requests,
+  or matching only by header. (#1753, @skriss)
+- Update route validation to comply with RFC-3986 "p-char" characters. (#1644,
+  @jackstine)
+- Illegal names like " " will be not allowed for query param name in
+  HTTPQueryParamMatch. (#1796, @gyohuangxin)
+* Webhook: Port is now considered when validating that ParentRefs are unique
+  (#1995, @howardjohn)
+
+### Conformance
```

Comment:
Do we need a release note for https://github.com/kubernetes-sigs/gateway-api/pull/2017? Maybe not because it's just a bug/reliability fix? WDYT?

Reply:
Added, thanks!